File: 数据结构与算法/极客时间/队列/实现/队列.py
class Queue:
    def __init__(self):
        self.val = [];
    
    # 入队
    def enqueue(self, item):
        self.val.append(item)

    # 出队
    def dequeue(self):
        front = self.front()
        self.val.remove(front)
        return front

    # 队首元素
    def front(self):
        return self.val[0]

File: 数据结构与算法/极客时间/队列/实现/队列.js
class Queue {
    constructor(val = null) {
        this.val = []
    }
    // == 入队   
    enqueue(item){
        this.val.push(item);
    }
    // == 出队
    dequeue(){
        return this.val.shift();
    }
    // == 队首元素
    front(){
        return this.val[0];
    }
}
